haystack/document_stores/elasticsearch.py

```diff
@@ -272,13 +272,27 @@ def get_documents_by_id(
         headers: Optional[Dict[str, str]] = None,
     ) -> List[Document]:
         """
-        Fetch documents by specifying a list of text id strings. Be aware that passing a large number of ids might lead
-        to performance issues. Note that Elasticsearch limits the number of results to 10,000 documents by default.
+        Fetch documents by specifying a list of text id strings.
+
+        :param ids: list of document IDs. Be aware that passing a large number of ids might lead to performance issues.
```

Comment:
Let's capitalize the beginning of argument descriptions (i.e. "List" instead of "list"). Can we give the user a sense of what a large number of ids is? Is 10K ok? 100K? Does it depend on how much is already indexed?

Reply:
Honestly I do not know.
This passage was already part of the original docstring.

Reply:
In my tests, I retrieved 17k documents with no particular issue.